service/ui.go

```diff
@@ -109,6 +111,17 @@ func NewUIServer(env evergreen.Environment, queue amboy.Queue, home string, fo T
 		hostCache: make(map[string]hostCacheItem),
 	}
 
+	if settings.AuthConfig.Kanopy != nil {
```

Comment:
Just to check my understanding, does this mean that this kind of validation  will only end up being used for staging environments?

Reply:
Yes. Though in the future, when the app is behind CorpSecure, we could use it for prod too if we wanted to skip the double auth.